infrastructure/sentry-subgraph/src/node-license.ts

```diff
@@ -8,6 +8,11 @@ import {
 } from "../generated/schema"
 
 export function handleTransfer(event: TransferEvent): void {
+  // We need to ignore keys greater than the totalSupply on deploy of the TK airdrop, so they won't get picked up by the operator
+  if (event.params.tokenId.gt(BigInt.fromI32(35180))) {
+    return;
+  }
+
```

Comment:
Will this break admin transfers in the sub-graph?

Reply:
It will not break it it will just generally ignore any keyIds above minted before TK release